.gitignore

```diff
@@ -1,5 +1,8 @@
+*.json
+*.html
 *.dat
 *.lock
+!dvc.lock
```

Comment:
Do we need this? If I delete it OC157 etc. it doesn't seem to make a difference to running things?